### PR TITLE
Add new mercutio keymap and update readme.md for layouts

### PR DESCRIPTION
**Pull request by Jonavin**
Add new mercutio keymap

## Description

- Add new mechwild/mercutio custom keymap with layered numpad and nav cluster
- keebio/quefency jonavin keymap layout changes  
- add README.md for joanvin keymaps
- numpad layers - left spacebar as Backspace

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/quefrency/keymaps/jonavin/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Jonathan's Quefrency layout - No Macros 65%
```
Comment (drashna) [inline keyboards/keebio/quefrency/keymaps/jonavin/README.md:1]: please rename this to be all lower case

File: keyboards/mechwild/mercutio/keymaps/jonavin/config.h [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Define custom font */
+#define OLED_FONT_H "keyboards/mechwild/mercutio/lib/mercutiofont.c"
```
Comment (drashna) [inline keyboards/mechwild/mercutio/keymaps/jonavin/config.h:21]: Should only need 
```suggestion
#define OLED_FONT_H "lib/mercutiofont.c"
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/keebio/quefrency/keymaps/jonavin/README.md (hunk 1 above)
Comment (Jonavin) [reply keyboards/keebio/quefrency/keymaps/jonavin/README.md]: Done

**Review by Jonavin — COMMENTED** [review]
Suggested changes applied

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**